From 6a71a9c4dd908cc0047765c6765fbbd6336d7305 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinh.nguyen@intel.com>
Date: Wed, 30 Oct 2019 14:10:50 -0500
Subject: [PATCH 046/151] HSD #1707096314: remove duplicate mpu_ccu_clk entry

commit  ba7f2f64e9456030bbef288f7b3a497786ed904a from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Remove the duplicate mpu_ccu_clk entry and clean up the
include/dt-bindings/clock/agilex-clock.h

Signed-off-by: Dinh Nguyen <dinh.nguyen@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 include/dt-bindings/clock/agilex-clock.h | 43 ++++++++++++------------
 1 file changed, 21 insertions(+), 22 deletions(-)

diff --git a/include/dt-bindings/clock/agilex-clock.h b/include/dt-bindings/clock/agilex-clock.h
index 2f641b8dd015..4c2a5e80ea86 100644
--- a/include/dt-bindings/clock/agilex-clock.h
+++ b/include/dt-bindings/clock/agilex-clock.h
@@ -44,27 +44,26 @@
 
 /* Gate clocks */
 #define AGILEX_MPU_CLK			30
-#define AGILEX_MPU_CCU_CLK		31
-#define AGILEX_MPU_PERIPH_CLK		32
-#define AGILEX_L4_MAIN_CLK		33
-#define AGILEX_L4_MP_CLK		34
-#define AGILEX_L4_SP_CLK		35
-#define AGILEX_CS_AT_CLK		36
-#define AGILEX_CS_TRACE_CLK		37
-#define AGILEX_CS_PDBG_CLK		38
-#define AGILEX_CS_TIMER_CLK		39
-#define AGILEX_S2F_USER0_CLK		40
-#define AGILEX_EMAC0_CLK		41
-#define AGILEX_EMAC1_CLK		43
-#define AGILEX_EMAC2_CLK		44
-#define AGILEX_EMAC_PTP_CLK		45
-#define AGILEX_GPIO_DB_CLK		46
-#define AGILEX_NAND_CLK			47
-#define AGILEX_PSI_REF_CLK		48
-#define AGILEX_S2F_USER1_CLK		49
-#define AGILEX_SDMMC_CLK		50
-#define AGILEX_SPI_M_CLK		51
-#define AGILEX_USB_CLK			52
-#define AGILEX_NUM_CLKS			53
+#define AGILEX_MPU_PERIPH_CLK		31
+#define AGILEX_L4_MAIN_CLK		32
+#define AGILEX_L4_MP_CLK		33
+#define AGILEX_L4_SP_CLK		34
+#define AGILEX_CS_AT_CLK		35
+#define AGILEX_CS_TRACE_CLK		36
+#define AGILEX_CS_PDBG_CLK		37
+#define AGILEX_CS_TIMER_CLK		38
+#define AGILEX_S2F_USER0_CLK		39
+#define AGILEX_EMAC0_CLK		40
+#define AGILEX_EMAC1_CLK		41
+#define AGILEX_EMAC2_CLK		42
+#define AGILEX_EMAC_PTP_CLK		43
+#define AGILEX_GPIO_DB_CLK		44
+#define AGILEX_NAND_CLK			45
+#define AGILEX_PSI_REF_CLK		46
+#define AGILEX_S2F_USER1_CLK		47
+#define AGILEX_SDMMC_CLK		48
+#define AGILEX_SPI_M_CLK		49
+#define AGILEX_USB_CLK			50
+#define AGILEX_NUM_CLKS			51
 
 #endif	/* __AGILEX_CLOCK_H */
-- 
2.26.1

